packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/chat.tests.js

```diff
@@ -319,6 +320,46 @@ QUnit.module('Chat', moduleConfig, () => {
             assert.strictEqual(getMessageGroups().length, 1);
         });
 
+        QUnit.test('Message Group should rendered in the scrollable content', function(assert) {
```

Comment:
All these tests relate to the messageList component. Let's move it to correspond test file.

Reply:
[d2c631211da8af6d62c8b2788168dde7ea574415]